content/300-guides/050-database/100-developing-with-prisma-migrate/170-customizing-migrations.mdx

```diff
@@ -30,9 +38,7 @@ To edit a migration file before applying it:
    npx prisma migrate dev
    ```
 
-</TopBlock>
-
-## Renaming a field
+### Example: Rename a field
```

Comment:
```suggestion
## Examples

### Example: Rename a field
```

Reply:
I prefer these as subsections of the general 'how to edit topic'